server/repository/migrations/postgres/2021-09-05T10-00_create_item_table/up.sql

```diff
@@ -1,11 +1,13 @@
 -- Create item table.
 
-CREATE TABLE item (
+CREATE TABLE item
+(
     id TEXT NOT NULL PRIMARY KEY,
     name TEXT NOT NULL,
     code TEXT NOT NULL,
     unit_id TEXT REFERENCES unit(id),
     type TEXT NOT NULL,
+    default_pack_size INTEGER NOT NULL,
```

Comment:
I went with `NOT NULL` since mSupply enforces a value for `default_pack_size` and it does make the reading of data easier, not needing to handle `Option` 🤷